.github/workflow/linters.yml

```diff
@@ -0,0 +1,100 @@
+Skip to content
```

Comment:
- [ ] Linter should be set up correctly for this repository, as for now, the linters check doesn't work on Github, kindly fix this